src/Settings.php

```diff
@@ -191,6 +191,13 @@ public function extend_v2_settings( $settings ) {
 			),
 			'type'        => 'kp_section_start',
 		);
+		$settings['onsite_messaging_enabled']          = array(
+			'title'       => __( 'Enable/Disable', 'klarna-onsite-messaging-for-woocommerce' ),
+			'label'       => __( 'Enable On-site Messaging', 'klarna-onsite-messaging-for-woocommerce' ),
+			'type'        => 'checkbox',
+			'description' => '',
+			'default'     => 'no',
```

Comment:
Change the default to yes maybe? Since we have customers that are already active with this and an update could make them disable this without knowing.